web/content/docs/benchmarks/elliptic/groundwater-flow-dirichlet-volumetric-source-term.pandoc

```diff
@@ -0,0 +1,107 @@
++++
+date = "2018-09-25T08:17:39+01:00"
+title = "Groundwater Flow (Volumetric Source Term)"
+project = "Elliptic/square_1x1_GroundWaterFlow/square_1e2_volumetricsourceterm.prj"
+author = "Tom Fischer"
+weight = 102
+
+aliases = [ "/docs/benchmarks/" ] # First benchmark page
+
+[menu]
+  [menu.benchmarks]
+    parent = "elliptic"
+
++++
```

Comment:
The test is OK and nicely documented, but for an actual user it might be more instructive to have some physical quantities with some actual units.

Reply:
You are right. I'll ask @waltherm or @Thomas-TK for a more practical test.